docs/usermanual/userman_admin.rst

```diff
@@ -446,9 +446,8 @@ After registering your device:
 
 .. screenshot::
     :user-role: admin
-    :url: /management/zone/
+    :url: /update/videos/
     :navigation-steps:
-    :focus: li.video
     :class: screenshot
```

Comment:
I was thinking of adding alternative text to all the screenshots in the docs, to improve accessibility. Since `screenshot` class inherits from `image`, can I add just this at the end of each?

`:alt: alternate text`  

@MCGallaspy